src/Makefile.am

```diff
@@ -662,6 +662,11 @@ libbitcoin_common_a_SOURCES = \
   script/standard.cpp \
   warnings.cpp \
   $(BITCOIN_CORE_H)
+
+if USE_LIBEVENT
+libbitcoin_common_a_CPPFLAGS += $(EVENT_CFLAGS) $(EVENT_PTHREADS_CFLAGS)
```

Comment:
Why `$(EVENT_PTHREADS_CFLAGS)` is here?

Reply:
Removed. There's no `evthread_` usage in the common lib. It's all in the node lib.